README.md

```diff
@@ -56,6 +56,7 @@ The following settings can currently be customized
 * allowDuplicates - defaults to **false**
 * caseSensitive - defaults to **true**
 * readOnly - defaults to **false**
+* singleFieldDelimiter - defaults to **,**
```

Comment:
Can also add the property in the example below:
`
[CultureSpecific]
[UIHint("Tags")]
[GetaTags(AllowSpaces = true, AllowDuplicates = true, CaseSensitive = false, ReadOnly = true)]
public virtual string Tags { get; set; }`